src/lib/openapi/index.ts

```diff
@@ -233,207 +34,19 @@ interface OpenAPIV3DocumentWithServers extends OpenAPIV3.Document {
-    profileSchema,
-    projectApplicationSchema,
-    projectApplicationSdkSchema,
-    projectApplicationsSchema,
-    projectEnvironmentSchema,
-    projectSchema,
-    projectsSchema,
-    proxyClientSchema,
-    proxyFeatureSchema,
-    proxyFeaturesSchema,
-    publicSignupTokenCreateSchema,
-    publicSignupTokenSchema,
-    publicSignupTokensSchema,
-    publicSignupTokenUpdateSchema,
-    pushVariantsSchema,
-    projectStatsSchema,
-    resetPasswordSchema,
-    requestsPerSecondSchema,
-    requestsPerSecondSegmentedSchema,
-    roleSchema,
-    rolesSchema,
-    sdkContextSchema,
-    sdkFlatContextSchema,
-    searchEventsSchema,
-    segmentSchema,
-    setStrategySortOrderSchema,
-    setUiConfigSchema,
-    sortOrderSchema,
-    splashRequestSchema,
-    splashResponseSchema,
-    stateSchema,
-    strategiesSchema,
-    strategySchema,
-    tagsBulkAddSchema,
-    tagSchema,
-    tagsSchema,
-    tagTypeSchema,
-    tagTypesSchema,
-    tagWithVersionSchema,
-    tokenUserSchema,
-    tokenStringListSchema,
-    uiConfigSchema,
-    updateApiTokenSchema,
-    updateFeatureSchema,
-    updateFeatureStrategySchema,
-    updateTagTypeSchema,
-    updateUserSchema,
-    updateTagsSchema,
-    createContextFieldSchema,
-    updateContextFieldSchema,
-    upsertSegmentSchema,
-    createStrategySchema,
-    updateStrategySchema,
-    updateFeatureTypeLifetimeSchema,
-    userSchema,
-    createUserResponseSchema,
-    usersGroupsBaseSchema,
-    usersSchema,
-    usersSearchSchema,
-    validatedEdgeTokensSchema,
-    validateFeatureSchema,
-    validatePasswordSchema,
-    validateTagTypeSchema,
-    variantSchema,
-    variantFlagSchema,
-    variantsSchema,
-    versionSchema,
-    deprecatedProjectOverviewSchema,
-    importTogglesSchema,
-    importTogglesValidateSchema,
-    importTogglesValidateItemSchema,
-    contextFieldStrategiesSchema,
-    telemetrySettingsSchema,
-    strategyVariantSchema,
-    createStrategyVariantSchema,
-    clientSegmentSchema,
-    createGroupSchema,
-    createFeatureNamingPatternSchema,
-    doraFeaturesSchema,
-    projectDoraMetricsSchema,
-    segmentsSchema,
-    updateFeatureStrategySegmentsSchema,
-    dependentFeatureSchema,
-    createDependentFeatureSchema,
-    parentFeatureOptionsSchema,
-    featureDependenciesSchema,
-    dependenciesExistSchema,
-    validateArchiveFeaturesSchema,
-    searchFeaturesSchema,
-    featureTypeCountSchema,
-    projectOverviewSchema,
-    featureSearchResponseSchema,
-    inactiveUserSchema,
-    inactiveUsersSchema,
-    recordUiErrorSchema,
-    resourceLimitsSchema,
-};
+/*
+ * All schemas in `openapi/spec` should be listed here.
+ * Instead of listing them all maunally, exclude those that are not schemas (maybe they should be moved elsewhere)
+ */
+import * as importedSchemas from './spec';
+const {
+    constraintSchemaBase,
+    unknownFeatureEvaluationResult,
+    playgroundStrategyEvaluation,
+    strategyEvaluationResults,
```

Comment:
These are not schemas, but they are exported inside schema files. I don't think it's a problem but at least it posts a question on whether they should be there or not.

Reply:
Good catch. I think I'm responsible for all of these, actually 😅 I think some of them make sense to keep where they are, but others maybe not. I don't think `unknownFeatureEvaluationResult` is actually used anywhere 🤔